frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.tsx

```diff
@@ -1,51 +1,50 @@
-import { createContext, useCallback } from 'react';
-import { useTelemetry } from '@console/shared/src/hooks/useTelemetry';
-import { useUserSettings } from '@console/shared/src/hooks/useUserSettings';
-import { QUICKSTART_REDUX_STATE_LOCAL_STORAGE_KEY, QUICKSTART_TASKS_INITIAL_STATES } from './const';
+import * as React from 'react';
 import {
-  AllQuickStartStates,
-  QuickStartState,
+  QuickStartContext,
+  QuickStartContextProvider,
+  QuickStartContextValues,
+  getDefaultQuickStartState,
   QuickStartStatus,
   QuickStartTaskStatus,
-} from './quick-start-types';
-import { getTaskStatusKey } from './quick-start-utils';
+  getTaskStatusKey,
+  QUICKSTART_TASKS_INITIAL_STATES,
+} from '@patternfly/quickstarts';
+import Pseudo from 'i18next-pseudo';
+import { useTranslation } from 'react-i18next';
+import {
+  MarkdownExecuteSnippet,
+  useInlineExecuteCommandShowdownExtension,
+  useMultilineExecuteCommandShowdownExtension,
+} from '@console/shared';
+import { useTelemetry } from '@console/shared/src/hooks/useTelemetry';
+import { useUserSettings } from '@console/shared/src/hooks/useUserSettings';
 
-export type QuickStartContextValues = {
-  activeQuickStartID: string;
-  allQuickStartStates?: AllQuickStartStates;
-  activeQuickStartState?: QuickStartState;
-  setActiveQuickStart?: (quickStartId: string, totalTasks?: number) => void;
-  startQuickStart?: (quickStartId: string, totalTasks: number) => void;
-  restartQuickStart?: (quickStartId: string, totalTasks: number) => void;
-  nextStep?: (totalTasks: number) => void;
-  previousStep?: () => void;
-  setQuickStartTaskNumber?: (quickStartId: string, taskNumber: number) => void;
-  setQuickStartTaskStatus?: (taskStatus: QuickStartTaskStatus) => void;
-  getQuickStartForId?: (id: string) => QuickStartState;
-};
+export { QuickStartContext };
+export { QuickStartContextProvider };
 
-export const QuickStartContext = createContext<QuickStartContextValues>({ activeQuickStartID: '' });
+export const getProcessedResourceBundle = (resourceBundle, lng) => {
+  const params = new URLSearchParams(window.location.search);
+  const pseudolocalizationEnabled = params.get('pseudolocalization') === 'true';
+
+  const language = lng || localStorage.getItem('bridge/language') || 'en';
+  const consoleBundle = resourceBundle;
+  if (pseudolocalizationEnabled && language === 'en') {
+    const pseudo = new Pseudo({ enabled: true, wrapped: true });
+    Object.keys(consoleBundle).forEach((key) => {
+      consoleBundle[key] = pseudo.process(consoleBundle[key], '', {}, { language });
+    });
+  }
+
+  return consoleBundle;
```

Comment:
Do not mutate the `resourceBundle` because it's causing the pseudo localization to stack.
```suggestion
  let consoleBundle = resourceBundle;
  if (pseudolocalizationEnabled && language === 'en') {
    consoleBundle = {};
    const pseudo = new Pseudo({ enabled: true, wrapped: true });
    Object.keys(resourceBundle).forEach((key) => {
      consoleBundle[key] = pseudo.process(resourceBundle[key], '', {}, { language });
    });
  }

  return consoleBundle;
```